src/controllers/Flashcards.js

```diff
@@ -0,0 +1,99 @@
+const Flashcard = require('../models/Flashcard');
+const { StatusCodes } = require('http-status-codes');
+
+const getAllFlashcards = async (req, res) => {
+    const flashcards = await Flashcard.find({}).sort('createdAt');
+    res.status(StatusCodes.OK).json({ flashcards, count: flashcards.length });
+};
+
+const getUserFlashcards = async (req, res) => {
+    const flashcards = await Flashcard.find({ createdBy: req.user.userId }).sort('createdAt');
```

Comment:
[Optional]
No need to fix this right away if you don't want to. Just add another card in JIRA for us to fix it eventually.

[Pagination]
Just like above, if a user had tons of flashcards this would be a problem.

Reply:
added pagination as recommended